.github/workflows/build.yml

```diff
@@ -11,12 +11,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["3.0", 3.1, 3.2, 3.3]
+        ruby: ["3.0", 3.1, 3.2, 3.3, 3.4]
         rails: [rails_6, rails_6_1, rails_7, rails_7_1, rails_7_2]
         twilio: [twilio_6, twilio_7]
         exclude:
           - rails: rails_7_2 # Rails 7.2 requires Ruby 3.1
             ruby: "3.0"
+          - rails: rails_6 # minitest from this version doesn't supporter Ruby 3.4
```

Comment:
@wwahammy If minitest from these specific versions do not support Ruby 3.4, then why update the ruby version underneath each line to show Ruby 3.4? 
Is this a workaround until minitest updates to support Ruby 3.4?

Reply:
@camckin10 this is under the "exclude" property, which describes matrix combinations that should be excluded that would otherwise be run.

The issue isn't that no version of minitest supports Ruby 3.4, many versions do. The problem is that the ones that are dependencies of Rails 6, 6.1 and 7 don't support Ruby 3.4.  Logically, those versions of Rails will never be updated to use later versions of minitest, since they're out of support. But we still want to confirm that this code works in as many possible combinations of Rails version, Twilio gem version and Ruby versions as we can.